rules/CodeQuality/Rector/Foreach_/ForeachItemsAssignToEmptyArrayToAssignRector.php

```diff
@@ -115,21 +115,21 @@ private function shouldSkip(Foreach_ $foreach): bool
             return true;
         }
 
-        /** @var Scope $scope */
         $scope = $foreach->expr->getAttribute(AttributeKey::SCOPE);
-
         if (! $scope instanceof Scope) {
-            return false;
+            return true;
```

Comment:
it came from https://github.com/rectorphp/rector-src/pull/735, no scope should be skipped.